### PR TITLE
Allow actions to know when they were last generated

### PR DESCRIPTION
**Pull request by apmasell**
Since some actions should change their behaviour if they are stale, this allows actions to know that. This information can't be used alone since `.actnow` actions will quickly become stale. This adds a separate policy to Vidarr that allows determining the desired submission behaviour to prevent stale actions from submitting.

JIRA Ticket: 

- [ ] Updates Changelog
- [ ] Updates developer documentation

### CODE REVIEW

File: plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/SubmitAction.java [hunk 1]
```diff
@@ -149,11 +152,6 @@ public Stream<ActionCommand<?>> commands() {
     return state.canReattempt() ? Stream.of(DELETE, REATTEMPT, RESET) : Stream.of(RESET);
   }
 
-  @ActionParameter(name = "dry_run", required = false)
```
Comment (avarsava) [inline plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/SubmitAction.java:152]: Why is this removed? Pretty sure we use this option

**Review by avarsava — APPROVED** [review]

**Review by callunity — APPROVED** [review]
